index.js

```diff
@@ -72,6 +73,11 @@ module.exports = function (target, dest, opts) {
                 fetchArgs.push('--save');
             }
 
+            if (opts.save_exact) {
```

Comment:
So I think this implementation could allow both `--save` and `--save-exact` to be added to `fetchArgs` if `opts.save = true` and `opts.save_exact = true`. 

I think we should only have one or the other. 

```
 if (opts.save_exact) {
    events.emit('verbose', 'saving exact');
    fetchArgs.push('--save-exact');
} else if (opts.save) {
    events.emit('verbose', 'saving');
    fetchArgs.push('--save');
 }
```